sdv/metadata/multi_table.py

```diff
@@ -199,3 +200,57 @@ def detect_table_from_dataframe(self, table_name, data):
         table = SingleTableMetadata()
         table.detect_from_dataframe(data)
         self._tables[table_name] = table
+
+    def _validate_table_exists(self, table_name):
```

Comment:
there is a method called `_validate_table_name` above that does the same thing. You can go with this method name, but just move it up and use it everywhere that `_valdiate_table_name` is used

Reply:
I'll go with `_validate_table_exists` then, alright? It seems more descriptive to me.